apps/extension/src/containers/wallet-panel/staking/stake-modal/index.tsx

```diff
@@ -56,7 +56,7 @@ export const StakeModal: React.FC<IProps> = ({ trigger, tooltipMessage, validato
 
 	let amount = new BigNumber(0)
 	if (reduceStake) {
-		const staked = stakedPositions?.stakes.find(position => position.validator.toString() === validator.address)
+		const staked = stakedPositions?.stakes.find(position => position.validator.toString() === validator?.address)
```

Comment:
no need to iterate if validator is null
```suggestion
		const staked = validator ?stakedPositions?.stakes.find(position => position.validator.toString() === validator.address): null
```